setup.py

```diff
@@ -309,18 +309,10 @@ def get_long_description():
             "mypy",
             "setuptools",
             "Sphinx!=3.2.0",
-            # Python 3.9 exclusions:
-            #
-            # * NumPy installation fails on Python 3.9 on the Ubuntu Xenial
-            #   system used by Travis CI.
-            # * PySide2 is installable but also currently not working
-            #   on Python 3.9.
-            # * Because of the above, we also exclude GUI-using packages
-            #   Pyface and TraitsUI on Python 3.9.
-            "numpy;python_version<'3.9'",
-            "pyface;python_version<'3.9'",
-            "PySide2;python_version<'3.9'",
-            "traitsui;python_version<'3.9'",
+            "numpy",
```

Comment:
I tried this fairly recently, and still encountered issues with NumPy on Python 3.9-dev. But it may be worth trying again.